src/features/menu/components/ItemsList.tsx

```diff
@@ -1,14 +1,81 @@
-import React from 'react';
-import { FlatList, StyleSheet, View } from 'react-native';
+import React, { useEffect } from 'react';
+import { FlatList, StyleSheet, Text, View } from 'react-native';
 import { BKGD_GREY } from '../../../constants/colors';
 import * as Types from '../../../constants/types';
+import { useSelector } from 'react-redux';
+import * as reducer from '../reducers/categories.reducer';
+import * as Actions from '../reducers/categories.reducer';
+import * as Strings from '../../../constants/strings';
+import { Category, Link, OtherListItem } from '../../../constants/types';
+import MoveoSVG from '../../../../assets/svg/MoveoSVG';
+
+const ItemsList = () => {
+  const categories = useSelector(reducer.selectFilteredCategories);
+  const links = useSelector(reducer.selectLinks);
+
+  //Call Api Function when component first mounts
+  useEffect(() => {
+    Actions.getCategories();
+  }, []);
+
+  const separator: Types.OtherListItem = { title: Strings.GENERAL_ITEM };
+
+  const logo: Types.OtherListItem = { title: Strings.LOGO };
+
+  let combinedArr: Array<Category | OtherListItem | Link> = [...categories];
+
+  combinedArr.push(separator);
+
+  if (links) {
+    combinedArr.push({ title: Strings.PRIVACY_POLICY, link: links.privacy_policy });
+    combinedArr.push({ title: Strings.TERMS_OF_USE, link: links.terms_of_use });
+  }
+
+  combinedArr.push(logo);
+  function renderItem(item: { item: Types.ListItem }) {
+    const getListItemText = () => {
+      if ((item.item as Types.Category).title && item.item.title) {
+        return item.item.title;
+      } else {
+        if (item.item as Types.Link) {
+          return (item.item as Types.Link).title;
+        }
+      }
+    };
+    if (item.item && item.item.title === logo.title) {
```

Comment:
The best way to do this would be to use an enum for the options you have.
i.e:
```suggestion
    // In types.ts file set these types and use them here
export type Category = {
  id: number;
  isParentCategory: boolean;
  children: Category[];
}

export enum ListType {
  Item,
  Logo,
  Separator,
  Link
};

export interface IBaseItem {
  type: ListType;
};

export interface ITitleItem {
  title: string;
};

export interface ILinkItem {
  link: string;
}

export interface ICategoryItem {
  id: number;
  isParentCategory: boolean;
  children: Category[];
}

export type ListItem = IBaseItem & ITitleItem;

export type LinkItem = IBaseItem & ITitleItem & ILinkItem;

export type CategoryItem = IBaseItem & ITitleItem & ICategoryItem;
```